src/libsync/iconjob.h

```diff
@@ -27,14 +29,15 @@ namespace OCC {
  * @brief Job to fetch a icon
  * @ingroup gui
  */
-class IconJob : public QObject
+class OWNCLOUDSYNC_EXPORT IconJob : public QObject
```

Comment:
did you check the user agent sent by the `IconJob` ?

Reply:
Yes, it's mirall.